docs/en/ingest-management/commands.asciidoc

```diff
@@ -31,31 +31,90 @@ NOTE: You might need to log in as a root user to run these commands.
 [[elastic-agent-enroll-command]]
 == elastic-agent enroll
 
-Enroll the {elastic-agent} in {fleet}.
+Enroll the {agent} in {fleet}.
+
+Use this command to enroll the {agent} in {fleet} without installing the
+agent as a service. You will need to do this if you installed
+the {agent} from a DEB or RPM package and plan to use systemd commands to
+start and manage the service. This command is also useful for testing
+{agent} prior to installing it.
+
+If you've already installed {agent}, use this command to enroll the agent in a different policy. 
```

Comment:
@EricDavisX I think this captures the essence of what you indicated below, but might be inaccurate (didn't test). Can I just run the enroll command again, or do I need to first unenroll the agent in Fleet?
```suggestion
If you've already installed {agent}, use this command to modify the settings that {agent} runs with. 
```